torch/linalg/__init__.py

```diff
@@ -140,3 +140,33 @@
     >>> LA.norm(m[0, :, :]), LA.norm(m[1, :, :])
     (tensor(3.7417), tensor(11.2250))
 """)
+
+tensorsolve = _add_docstr(_linalg.linalg_tensorsolve, r"""
+linalg.tensorsolve(a, b, axes=None) -> Tensor
+
+Computes the solution to the tensor equation ``a x = b`` for ``x``.
+
+Suppose that `:attr:`b` is equivalent to ``torch.tensordot(a, x)``.
+This function computes tensor ``x`` from `:attr:`a` and `:attr:`b`.
+
+Args:
+    a (Tensor): left-hand-side tensor of size ``b.shape + Q``.
+        ``Q`` is the shape of that sub-tensor of `:attr:`a` consisting of the appropriate
+        number of its rightmost indices, and must be such that ``prod(Q) == prod(b.shape)``.
+    b (Tensor): right-hand-side tensor of any size.
```

Comment:
"solution to tensordot(:attr:\`input\`, x). May be any size."

Is that true that it may actually be any size? Will an empty tensor work, for example?

Reply:
Size == shape here.

It's true that can be any size. Choosing `b` (or `other`) puts restrictions on shape of `a` (or `input`).
An empty tensor would work if `lda` parameter (4th argument) in `lapackSolve` call was handled correctly.